config/routes.rb

```diff
@@ -1,13 +1,16 @@
 Rails.application.routes.draw do
+  root "photos#index"
+
+  devise_for :users
+
   resources :likes
   resources :follow_requests
   resources :comments
   resources :photos
-  devise_for :users
+  
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
```

Comment:
Can remove commented out lines if they are not relevant to your code

Reply:
Will do! Thank you!